pages/operations/uninstall.md

```diff
@@ -1,15 +1,26 @@
 ---
-title: Destroying the Cluster Safely
+title: Destroying the Cluster and Installations
 description: How do I bring things down safely?
 ---
 
 ## Overview
 
-Uninstalling any Plural application or an entire Plural installation is a one-liner in your terminal. If you want to delete your Plural installation, make sure to run `plural destroy` before deleting your Git repository. If you delete your Git repository first, you will have to manually clean up all the resources that Plural has provisioned for you.&#x20;
+Plural provides options to uninstall specific applications, tear down your clusters, and wipe any references to installations to start from zero. Uninstalling any Plural application or an entire Plural installation is a one-liner in your terminal or Cloud Shell. If you want to delete your Plural installation, make sure to run `plural destroy` before deleting your Git repository. If you delete your Git repository first, you will have to manually clean up all the resources that Plural has provisioned for you.&#x20;
+
+## Purging the Cloud Shell
+
+If you'd like to move your Cloud Shell instance locally, you can sync your shell and delete the Cloud Shell instance from our servers. To sync your shell and delete your current Cloud Shell instance, use:
```

Comment:
@michaeljguarino is there any other reason you'd want to purge your cloud shell? Like starting over from a partial setup?